examples/forum/.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__/*
```

Comment:
I think we actually want to put this in the root .gitignore. It should also match the __pycache__ directory in forum (in addition to any other __pycache__ directories in our project) if its put there.